og_migrate/src/Plugin/migrate/OgEntityDeriver.php

```diff
@@ -0,0 +1,302 @@
+          'message' => 'Migration entity is missing',
+        ],
+      ],
+      'entity_type' => [
+        [
+          'plugin' => 'og_entity_type_exists',
+          'source' => 'entity_type',
+          'bundle_property' => 'bundle',
+        ],
+        [
+          'plugin' => 'skip_on_empty',
+          'method' => 'row',
+          'message' => 'Entity type is missing',
+        ],
+      ],
+      'group_type' => [
+        [
+          'plugin' => 'og_entity_type_exists',
+          'source' => 'group_type',
+          'bundle_property' => 'bundle',
+        ],
+        [
+          'plugin' => 'skip_on_empty',
+          'method' => 'row',
+          'message' => 'Group type is missing',
+        ],
+      ],
+      'field_name' => [
+        'plugin' => 'static_map',
+        'source' => 'field_name',
+        'map' => [
+          'og_group_ref' => 'og_audience',
+          'default' => 'og_audience',
+        ],
+      ],
+      'language' => 'language',
+    ];
+  }
+
+  /**
+   * Gets the migration process definition for user.
+   *
+   * @param string $group_type_migration
+   *   The group_type migration id.
+   *
+   * @return array
+   *   The "process" definition.
+   */
+  protected function getUserProcessDefinition($group_type_migration) {
+    return [
+      'id' => 'id',
+      'type' => [
+        [
+          'plugin' => 'migration_lookup',
+          'migration' => 'd7_og_membership_type',
+          'source' => 'type',
+        ],
+        [
+          'plugin' => 'skip_on_empty',
+          'method' => 'row',
+          'message' => 'Membership type is missing',
+        ],
+      ],
+      'uid' => [
+        [
+          'plugin' => 'migration_lookup',
+          'migration' => 'd7_user',
+          'source' => 'etid',
+        ],
+        [
+          'plugin' => 'skip_on_empty',
+          'method' => 'row',
+          'message' => 'User is missing',
+        ],
+      ],
+      'entity_id' => [
+        [
+          'plugin' => 'migration_lookup',
+          'migration' => $group_type_migration,
+          'source' => 'gid',
+        ],
+        [
+          'plugin' => 'skip_on_empty',
+          'method' => 'row',
+          'message' => 'Migration group is missing',
+        ],
+      ],
+      'entity_type' => [
+        [
+          'plugin' => 'og_entity_type_exists',
+          'source' => 'group_type',
+          'bundle_property' => 'entity_bundle',
+        ],
+        [
+          'plugin' => 'skip_on_empty',
+          'method' => 'row',
+          'message' => 'Entity type is missing',
+        ],
+      ],
+      'roles' => [
+        'plugin' => 'migration_lookup',
+        'migration' => 'd7_og_role',
+        'source' => 'roles',
+      ],
+      'state' => [
+        'plugin' => 'static_map',
+        'source' => 'state',
+        'map' => [
+          '1' => 'active',
+          '2' => 'pending',
+          '3' => 'blocked',
+        ],
+        'default_value' => '1',
```

Comment:
Shouldn't this be `'default_value' => 'active'` ?

Reply:
Yes